natchez-datadog/src/main/scala/com/ovoenergy/effect/natchez/DatadogTags.scala

```diff
@@ -11,6 +11,8 @@ object DatadogTags {
   sealed trait SpanType
 
   object SpanType {
+    case object Custom extends SpanType
+    case object Cache extends SpanType
```

Comment:
I can see this type in the docs, but I'm not sure what it means. Any idea what it's supposed to be used for?

Reply:
I don't really know but my assumption is it is for profiling something like a Redis or Memcached cache

Reply:
yeah, does what it says on the tin =)